src/isomorphic/modern/element/__tests__/ReactJSXElementValidator-test.js

```diff
@@ -99,19 +100,49 @@ describe('ReactJSXElementValidator', () => {
       },
     };
 
-    void <Component>{iterable}</Component>;
+    ReactTestUtils.renderIntoDocument(<Component>{iterable}</Component>);
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toContain(
       'Each child in an array or iterator should have a unique "key" prop.',
     );
   });
 
+  it('does not warns for fragments of multiple elements without keys', () => {
+    spyOn(console, 'error');
+
+    ReactTestUtils.renderIntoDocument(
+      <React.Fragment>
+        <span>1</span>
+        <span>2</span>
+      </React.Fragment>,
+    );
+
+    expectDev(console.error.calls.count()).toBe(0);
```

Comment:
This is unnecessary as long as you remove both this *and* the `spyOn` call.
We assume tests shouldn't warn by default unless you `spyOn`.